packages/@glimmerx/babel-plugin-component-templates/test/fixtures/ember/output.js

```diff
@@ -0,0 +1,22 @@
+const {
+  _setComponentTemplate: _setComponentTemplate
```

Comment:
This seems pretty odd...

Reply:
I figured minifiers would take care of it, didn't seem like a huge deal